en/deploy-tiflash.md

```diff
@@ -79,6 +79,140 @@ To add TiFlash component to an existing TiDB cluster, you need to set `replicati
 
 If the server does not have an external network, refer to [deploy the TiDB cluster](deploy-on-general-kubernetes.md#deploy-the-tidb-cluster) to download the required Docker image on the machine with an external network and upload it to the server.
 
+## Remove TiFlash
+
+1. Adjust the number of replicas of the tables replicated to the TiFlash cluster.
+
+    To completely remove TiFlash, you need to set the number of replicas of all tables in TiFlash to `0`.
+
+    1. To connect to the TiDB service, refer to the steps in [Access the TiDB Cluster in Kubernetes](access-tidb.md).
```

Comment:
```suggestion
    1. Refer to the steps in [Access the TiDB Cluster in Kubernetes](access-tidb.md) to connect to the TiDB service.
```

Reply:
It's usually advised to be put the purpose before the task. See https://developers.google.cn/style/clause-order